previously was sed -i '/#define _LINUX_NETDEVICE_H/d' src/arpd/*.c in ebuild
--- a/src/arpd/arp.c
+++ b/src/arpd/arp.c
@@ -15,7 +15,6 @@
 #include <sys/types.h>
 #include <sys/socket.h> /* for linux/if_arp.h */
 #include <netinet/in.h> /* for ntohs, etc. */
-#define _LINUX_NETDEVICE_H /* very crude hack for glibc2 */
 #include <linux/types.h>
 #include <linux/if_arp.h>
 #include <linux/if_ether.h>
--- a/src/arpd/io.c
+++ b/src/arpd/io.c
@@ -21,7 +21,6 @@
 #include <atm.h>
 #include <linux/atmclip.h> /* for CLIP_DEFAULT_IDLETIMER */
 #include <linux/atmarp.h>
-#define _LINUX_NETDEVICE_H /* glibc2 */
 #include <linux/types.h>
 #include <linux/if_arp.h>
 
--- a/src/arpd/itf.c
+++ b/src/arpd/itf.c
@@ -12,7 +12,6 @@
 #include <sys/types.h>
 #include <linux/atmclip.h>
 #include <sys/socket.h>
-#define _LINUX_NETDEVICE_H /* glibc2 */
 #include <linux/types.h>
 #include <linux/if_arp.h>
 
